docs/guides/plans-overview.mdx

```diff
@@ -31,8 +31,13 @@ This plan is recommended for those who are performing quantum utility research p
 
 Run quantum circuits on the world's best QPUs using an enterprise quantum time subscription. Additionally, some functionality, such as [Qiskit Functions](functions) and [Qiskit Transpiler as a Service](qiskit-transpiler-service),  is available only to Premium Plan users.
 
-This plan is recommended for those who want to execute a strategic quantum roadmap and develop quantum algorithms and applications at scale.
+This plan is recommended for those who want to execute a strategic quantum roadmap and develop quantum algorithms and applications at scale. [Contact IBM](https://www.ibm.com/account/reg/us-en/signup?formid=MAIL-quantum) to learn more.
 
+## Dedicated Service
+
+Access to an entirely dedicated quantum system that is serviced and maintained by IBM Quantum.
+
```

Comment:
I think this isn't a complete sentence? I'm not sure I understand it.

Reply:
It's better now, but still seems a little confusing to me? I don't know what action I would take as a user. Maybe it would make for sense to combine the two paragraphs? 

Or maybe something like

> You can access an entirely dedicated quantum system

Or

> It is possible to access